### DISCUSSION TITLE
Merge Tree technical paper

### DISCUSSION THREAD

**Discussion by evanxg852000** [ClickHouse/ClickHouse#33732] [Q&A] [answered]
Hi, I have been hearing great things about ClickHouse Merge Tree technique. I have read the docs but I am wondering if there is any technical in-depth documentation or paper that I can read to understand how it's implemented.

**Comment by alexey-milovidov** [comment, marked as answer]
No, we don't have.

**Reply by Alex-Cheng** [reply]
I think code will tell you truth. Reading code.
